inc/ses_to_cloudwatch/namespace.php

```diff
@@ -45,21 +35,11 @@ function on_sent_message( $result, $message ) {
  * @param array $message The error message.
  */
 function on_error_sending_message( Exception $error, $message ) {
-	CloudWatch_Logs\send_events_to_stream(
-		[
-			[
-				'timestamp' => time() * 1000,
-				// phpcs:ignore WordPress.WP.AlternativeFunctions.json_encode_json_encode
-				'message'   => json_encode( [
-					'error'     => [
-						'class'   => get_class( $error ),
-						'message' => $error->getMessage(),
-					],
-					'message' => $message,
-				] ),
-			],
+	Cloud\get_logger( 'ses', 'Failed' )->error( json_encode( [
+		'error'     => [
+			'class'   => get_class( $error ),
+			'message' => $error->getMessage(),
```

Comment:
As above, this shouldn't be aligned.
```suggestion
		'error' => [
			'class' => get_class( $error ),
			'message' => $error->getMessage(),
```